simulation_ws/src/cloudwatch_simulation/launch/turtlebot3_navigation.launch

```diff
@@ -1,10 +1,10 @@
-<!-- Copied from turtlebot3_navigation, allow initial pose arguments -->                                                                                                            
+<!-- Copied from turtlebot3_navigation, allow initial pose arguments -->
 <launch>
   <!-- Arguments -->
-  <arg name="model" default="$(optenv TURTLEBOT3_MODEL waffle_pi)" doc="model type [burger, waffle, waffle_pi]"/>                                                                                
-  <arg name="map_file" default="$(find cloudwatch_simulation)/maps/map.yaml"/>
-  <arg name="open_rviz" default="true"/>
-  <arg name="initial_pose_x" default="0.0"/>                                                                                                                                        
+  <arg name="model" default="$(optenv TURTLEBOT3_MODEL waffle_pi)" doc="model type [burger, waffle, waffle_pi]"/>
+  <arg name="map_file" default="$(find turtlebot3_navigation)/maps/map.yaml"/>
```

Comment:
How was it working before? The `cloudwatch_simulation/maps/` folder looks empty.

Reply:
Yep it doesn't look right.. if run by itself `cloudwatch_simulation/launch/turtlebot3_navigation.launch` should fail but I suppose that is never done in our use-case

Reply:
Right, It didn’t work before